src/internal/useListeners/useListeners.ts

```diff
@@ -33,15 +33,35 @@ interface UseListenersResult {
 export function useListeners(): UseListenersResult {
   const globalListeners = useRef(new Map())
 
-  const addListener = useCallback((eventTarget, type, listener, options) => {
-    globalListeners.current.set(listener, { type, eventTarget, options })
-    eventTarget.addEventListener(type, listener, options)
-  }, [])
+  /* eslint-disable no-undef */
```

Comment:
why?

Reply:
because we use «magic» `AddEventListenerOptions` (there were same ignores before)